latest/Dockerfile.amd64

```diff
@@ -12,9 +12,13 @@ ARG BUILD_VERSION
 # renovate: datasource=github-releases depName=transifex/cli
 ENV TRANSIFEX_VERSION="${BUILD_VERSION:-v1.6.5}"
 
+# renovate: datasource=github-releases depName=owncloud-ci/gettext
+ENV GO_GETTEXT_VERSION="v1.0.0-beta.3"
```

Comment:
This syntax is not working. Renovate is using a RegexManager for this, and you have to match the defined regex.

Reply:
By default, pre-releases will be ignored by renovate.

Reply:
my plan was, to release 1.0.0 after merging this to test renovate. How would the regex look like?

Reply:
https://github.com/owncloud-ci/renovate-presets/blob/master/docker.json#L35

Just use the syntax from transifex above, e.g.

```
ARG GO_GETTEXT_VERSION

# renovate: datasource=github-releases depName=owncloud-ci/gettext
ENV GO_GETTEXT_VERSION="${GO_GETTEXT_VERSION:-v1.0.0-beta.3}"
```

Reply:
ah, i see https://github.com/owncloud-ci/renovate-presets/blob/master/docker.json#L35